packages/www/pages/dashboard/streams/[id]/index.tsx

```diff
@@ -1,20 +1,31 @@
-import StreamDetail from "layouts/streamDetail";
-import StreamSessionsTable from "components/Dashboard/SessionsTable";
-import MultistreamTargetsTable from "@components/Dashboard/MultistreamTargetsTable";
+import { useState, useCallback, useEffect } from "react";
 import { useRouter } from "next/router";
-import { Text } from "@livepeer.com/design-system";
-import { useCallback } from "react";
-import { useApi, useAnalyzer } from "hooks";
-import { Stream } from "@livepeer.com/api";
 import { useQuery, useQueryClient } from "react-query";
+import { Stream } from "@livepeer.com/api";
+import { useApi, useAnalyzer } from "hooks";
+import { StreamInfo } from "hooks/use-api";
+import StreamDetail from "layouts/streamDetail";
+import StreamSessionsTable from "@components/Dashboard/SessionsTable";
+import Logger from "@components/Dashboard/Logger";
+import Chart from "@components/Dashboard/Chart";
+import HealthChecksTable from "@components/Dashboard/HealthChecksTable";
+import MultistreamTargetsTable from "@components/Dashboard/MultistreamTargetsTable";
+import { Text, Box, Heading } from "@livepeer.com/design-system";
 
+const ingestInterval = 10 * 1000;
+const maxItems = 6;
 const refetchInterval = 5 * 1000;
 
 const Overview = () => {
   const router = useRouter();
   const queryClient = useQueryClient();
-  const { getStream } = useApi();
+  const { getStream, getStreamInfo } = useApi();
   const { getHealth } = useAnalyzer();
+  const [switchTab, setSwitchTab] = useState<"Overview" | "Health">("Overview");
```

Comment:
**nit:**
```suggestion
  const [currentTab, setCurrentTab] = useState<"Overview" | "Health">("Overview");
```
(or maye `setCurrentTab` could be `switchTab`, but I think `setSwitchTab` doesn't make sense)